### PR TITLE
Edited information concerning Zulu install

### LINKED ISSUE #451: Recommendation of Zulu
The [main installation page](http://docs.openhab.org/installation/index.html#prerequisites) mentions that Zulu's only disadvantage is that it's not available as a package for most systems default repos, but it's very simple to install in each:

 - [Deb based linux](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/PrepareZuluPlatform/AttachAPTRepositoryUbuntuOrDebianSys.htm)
 - [RPM based linux](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/PrepareZuluPlatform/AttachYumRepositoryRHEL-SLES-OracleLinuxSys.htm)
 - [Windows](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/InstallingZulu/InstallationWindowsUsingZuluMSIFile.htm)
 - Mac [GUI](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/InstallingZulu/InstallOnMacOSXUsingDMGPkgThroughGUI.htm) or [CLI](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/InstallingZulu/InstallOMacOSXUsingDMGPkgThroughCLI.htm)

Should we update the docs to use Zulu? Particularly for Linux, should we start detailing the instructions of Zulu instead of Webupd8's openjdk?

### PR DESCRIPTION
**Pull request by BClark09**
Closes #451

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

File: installation/linux.md [hunk 1]
```diff
@@ -29,51 +29,8 @@ Train your understanding of Linux permissions at [linuxjourney.com/lesson/file-p
...
-You can then install the downloaded package with:
-
-```shell
-sudo yum localinstall jdk-8u60-linux-x64.rpm
-```
-
-{% include collapsible/item-end.html %}
-{% include collapsible/end.html %}
+You may want to install Zulu [as a package](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/PrepareZuluPlatform/AttachAzulPackageRepositories.htm) or [manually](http://zulu.org/zuludocs-folder/Content/ZuluInstallationGuide/InstallingZulu/InstallLinuxUsingZuluZIPFile.htm).
+Alternatively, Zulu embedded can be installed for small systems either from the same package repository as above or [manually](http://www.azul.com/downloads/zulu-embedded/).
```
Comment (ThomDietrich) [inline installation/linux.md]: In light of http://zulu.org/zulu-9-pre-release-downloads you might want to specifically mention Zulu 8 / Zulu Embedded 8 here.

Also, it's called "Zulu Embedded" 😃 

Thanks!

Comment (BClark09) [inline installation/linux.md]: Eek, good point! I've just seen zulu-9 pop up on my install list!

**Comment by BClark09** [comment]
Added a bit about java alternatives.

**Comment by ThomDietrich** [comment]
Nice. Thanks!

**Comment by BClark09** [comment]
```
git checkout avoid-thomas-wrath
git commit -s -m "Write future commit messages in imperative mood."
```

**Pull request merged**